### ISSUE TITLE
Why primeflex responsive is inverted? #195

### ISSUE THREAD

**Issue by samuel-aka-viana** [primefaces/primeflex#197]
i have a exemple of code, """<div class="flex align-content-center md:align-content-start flex-wrap" style="min-height: 200px; background: red">
  <div class="flex align-items-center justify-content-center w-4rem h-4rem bg-primary font-bold border-round m-2"> abacatada 1
  </div>
  <div class="flex align-items-center justify-content-center w-4rem h-4rem bg-primary font-bold border-round m-2">abacatada 2
  </div>
  <div class="flex align-items-center justify-content-center w-4rem h-4rem bg-primary font-bold border-round m-2">abacatada 3
  </div>
  <div class="flex align-items-center justify-content-center w-4rem h-4rem bg-primary font-bold border-round m-2">abacatada 4
  </div>
  <div class="flex align-items-center justify-content-center w-4rem h-4rem bg-primary font-bold border-round m-2">abacatada 5
  </div>
</div>""" but when i try to use the responsive, the clausule md:align-content-start has preference in code, begin with center, not using start  when as i declared align-content-center

**Comment by nharrer** [comment]
Yes, the same here. Maybe I am misunderstanding the documentation. Shouldn't `sm:xxxx` apply `xxxx` to small screens only? But it seems to be the opposite.

Example with primeflex 3.3.1: 
```
<div class="flex flex-row sm:flex-column gap-2" style="background-color: red">
  <div>aaaa</div>
  <div>bbbb</div>
  <div>cccc</div>
</div>
```

I would expect to have a `flex-direction` of `column` for screens <= 576px. And a `row` layout otherwise. But it's just the other way around:

![image](https://github.com/primefaces/primeflex/assets/8344095/2a2bd0cb-722f-49de-b7e0-181428e48adb)
![image](https://github.com/primefaces/primeflex/assets/8344095/05102ca1-f020-48c2-b925-e1186b4d3f7b)
Example on Stackblitz: https://stackblitz.com/edit/angular-ivy-khexpu

**Comment by nharrer** [comment]
Please ignore my last comment. Coming from Angular Flex-Layout, that seemed to be the way it was supposed to work.

But in fact, it is a mobile-first breakpoint system. I only realized that after learning to use Tailwind and studying their [documentation](https://tailwindcss.com/docs/responsive-design#working-mobile-first):

> Where this approach surprises people most often is that to style something for mobile, you need to use the unprefixed version of a utility, not the sm: prefixed version. Don’t think of sm: as meaning “on small screens”, think of it as “at the small breakpoint“.

So `sm:` means at the breakpoint `sm` and above. It makes sense now, and everything is working as intended.

However, I have a little critique: The PrimeFlex documentation doesn't mention that at all. So for people some people with a different background, this can be a bit confusing.
